app/helpers/claims/show_helper.rb

```diff
@@ -11,5 +11,21 @@ def claim_submitted_title(claim)
     def shared_view_css_class_size(claim)
       claim.has_ecp_policy? ? "l" : "xl"
     end
+
+    def base_and_uplift_award_amounts(claim, year)
```

Comment:
Seems like this should be named after an object that it returns, ie, missing object.